include/base/sparsity_pattern.h

```diff
@@ -80,8 +100,57 @@ static void sort_row (const BidirectionalIterator begin,
  */
 class Build : public ParallelObject
 {
+public:
+  Build (const DofMap & dof_map_in,
+         const CouplingMatrix * dof_coupling_in,
+         const std::set<GhostingFunctor *> & coupling_functors_in,
+         const bool implicit_neighbor_dofs_in,
+         const bool need_full_sparsity_pattern_in);
+
+  Build (Build & other, Threads::split);
```

Comment:
Consider defaulting the move constructor/deleting the other special functions for `Build` class? Not really necessary, but I feel it's good to have simply as documentation.